### PR TITLE
Update English.php

### PR DESCRIPTION
**Pull request by yesszus**
Issue: #3493

### CODE REVIEW

**Comment by tgtje** [comment]
@yesszus  would it not be better to change the English define in the plugin itself? , instead of making it site wide ? Pages is also in use in social... and as word also in e_sitelink in /page.
Just to not 'confuse' the usage of the word 'Pages'.  so it becomes more of a 'function use' just for search only?

File: e107_languages/English/English.php [hunk 1]
```diff
@@ -142,5 +142,5 @@
 define("LAN_REQUIRED_BLANK", "Required field(s) were left blank.");
 define("LAN_PLEASEWAIT", "Please Wait");
 define("LAN_CHOOSE_FILE", "Choose a file");
-
+define("LAN_PAGES", "Pages");
```
Comment (CaMer0n) [inline e107_languages/English/English.php]: Not here. Should be in e107_plugins/page/languages/English/English_global.php
```
define("LAN_PLUGIN_PAGE_NAME", "Pages");
```

**Comment by yesszus** [comment]
@CaMer0n changed this.

**Comment by tgtje** [comment]
👍  think this way is better ( eg. in overall 'understanding' of a Title versus a Result)
Nice work anyway

**Comment by CaMer0n** [comment]
Thank you @yesszus @tgtje

**Pull request merged**